src/yunohost/settings.py

```diff
@@ -90,9 +100,12 @@ def settings_set(key, value):
                 received_type=type(value).__name__, expected_type=key_type))
     elif key_type == "int":
         if not isinstance(value, int) or isinstance(value, bool):
```

Comment:
Not sure to understand this condition ... if `value` is a `bool`, it's not an `int` anyway ? So the second condition is useless ? Or am I too sleepy ?